### PR TITLE
Add emptydir extension

### PR DESCRIPTION
**Pull request by skonto**
## Proposed Changes
- Adds doc under feature flags page for empty dir. This is part of [this work](https://github.com/knative/serving/issues/11664).

### CODE REVIEW

File: docs/serving/feature-flags.md [hunk 1]
```diff
@@ -106,6 +106,29 @@ spec:
           image: gcr.io/knative-samples/helloworld-java
 ```
 
+## EmptyDir
+* **Type**: extension
+* **ConfigMap key:** `kubernetes.podspec-volumes-emptydir`
+
+This extension controls whether [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) can be specified.
```
Comment (julz) [inline docs/serving/feature-flags.md:113]:
```suggestion
This extension controls whether [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) volumes can be specified.
```

**Review by julz — COMMENTED** [review]
🎉

File: docs/serving/feature-flags.md [hunk 2]
```diff
@@ -106,6 +106,29 @@ spec:
           image: gcr.io/knative-samples/helloworld-java
 ```
 
+## EmptyDir
+* **Type**: extension
+* **ConfigMap key:** `kubernetes.podspec-volumes-emptydir`
```
Comment (abrennan89) [inline docs/serving/feature-flags.md:111]:
```suggestion
## EmptyDir

* **Type**: extension
* **ConfigMap key:** `kubernetes.podspec-volumes-emptydir`
```

**Review by abrennan89 — COMMENTED** [review]
one very minor formatting tweak to add a line space

File: docs/serving/feature-flags.md (hunk 2 above)
Comment (abrennan89) [reply docs/serving/feature-flags.md:111]: not important :)

**Pull request merged**